var/spack/repos/builtin/packages/py-moarchiving/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMoarchiving(PythonPackage):
+    """
+    Biobjective Archive class with hypervolume indicator
+    and uncrowded hypervolume improvement computation
+    """
+
+    homepage = "https://github.com/CMA-ES/moarchiving"
+    pypi = "moarchiving/moarchiving-0.6.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("0.6.0", sha256="705ded992d399bc1ac703e68391bded6f64e1bde81b2bb25061eaa6208b5b29a")
+
+    variant("arbitrary_precision", default=True, description="Build with Fraction support")
```

Comment:
Is there a reason why you have renamed the variant? In `setup.py` a `-` is used instead of a `_` (meaning it's called `arbitrary-precision`).
@adamjstewart do we have a policy for that?

And any reason why it should be enabled by default?

Reply:
"-" is used by spack in the same way of "~" so I prefered to name the variant `arbitrary_precision`

Reply:
I also prefer underscores over dashes, but we don't have a strict policy

Reply:
I usually default to False by default to match `pip install` behavior and minimize build times.